src/ringbuffer.toit

```diff
@@ -15,24 +15,39 @@ class RingBuffer:
   append value/float:
     buffer[head] = value
     head = (head + 1) % size_
-    count = min (count + 1) size_
+    count = (min (count + 1) size_)
```

Comment:
why you put brackets here? it is a method call, it is not really needed I think.

Reply:
no need, delete